tests/api2/test_530_ups.py

```diff
@@ -28,25 +36,92 @@
     'monpwd'
 ]
 
+default_dummy_data = {
+    'battery.charge': 100,
+    'driver.parameter.pollinterval': 2,
+    'input.frequency': 49.9,
+    'input.frequency.nominal': 50.0,
+    'input.voltage': 230,
+    'input.voltage.nominal': 240,
+    'ups.status': 'OL',
+    'ups.timer.shutdown': -1,
+    'ups.timer.start': -1,
+}
+
+
+def remove_file(filepath):
+    ssh(f'rm {filepath}', check=False)
+
+
+def did_shutdown():
+    return ssh(f'cat {SHUTDOWN_MARKER_FILE}', check=False) == "done\n"
+
+
+def write_fake_data(data={}):
```

Comment:
data=None and then inside function do `all_data = default_dummy_data | data or {}`